editor/test/resources/save_data_project/checked.md

```diff
@@ -0,0 +1,16 @@
+# Title
+Title paragraph text.
+
+## Heading
+Heading paragraph text.
+
+### Sub-heading
+Sub-heading paragraph text.
+
+* Unordered
+* List
+
+---
+
+1. Ordered
+2. List
```

Comment:
I added this because of  `save_data_test/all-resource-types-covered-test` and fixed the tests under `save_test.clj`. Is there any additional action needed?

Reply:
I think that's enough for tests :)

Reply:
That should do it! ❤️